### PR TITLE
Improve comment about what the '**' glob pattern does.

### PR DESCRIPTION
**Pull request by chaitanyagurrapu**
Improve comment about what the '**' glob pattern does.

### CODE REVIEW

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -246,7 +246,7 @@ build/
 # ignore doc/notes.txt, but not doc/server/arch.txt
 doc/*.txt
 
-# ignore all .pdf files in the doc/ directory
+# ignore all .pdf files in the doc/ directory or in the sub directories of doc/
```
Comment (ben) [inline book/02-git-basics/sections/recording-changes.asc]: Hmm. What about this instead?

> `# ignore all .pdf files anywhere inside the doc/ directory`

Comment (chaitanyagurrapu) [inline book/02-git-basics/sections/recording-changes.asc]: I think it is nicer to have the word "sub directory" in there somewhere. To make the association with '**' stronger and remove any ambiguity. After all that is what it is all about, which you might not know if you were seeing it for the first time.

Comment (ben) [inline book/02-git-basics/sections/recording-changes.asc]: OK, how about this then:

> `# ignore all .pdf files in the doc/ directory or any of its subdirectories`

Comment (mdkalish) [inline book/02-git-basics/sections/recording-changes.asc]: Maybe this is logically "more true"? :D

> \# ignore all .pdf files in the doc/ directory ~or~ and any of its subdirectories

**Comment by chaitanyagurrapu** [comment]
Updated accordingly.

**Comment by ben** [comment]
👍 Thanks!

**Pull request merged**